### PR TITLE
Add DEVELOPMENT.md

### PR DESCRIPTION
**Pull request by abdulmth**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Run OpenDID without Docker
+
+To run the OpenDID locally,
+
+1. Run the setup or add your `config.yaml` file with your seed.
+
+```bash
+ENDPOINT=peregrine ./scripts/setup.sh "build hill second flame trigger simple rigid cabbage phrase evolve final eight"
```
Comment (ggera) [inline DEVELOPMENT.md:8]: whose seed is this? May be some note how to bump it from faucet. I think it's better if every dev uses their own peregrine account

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+4. In a different terminal, run the demo project.
+
+```bash
+cd demo-project && yarn && yarn build && node index.js
+```
+
+## Release Process
+
+Releasing if trigged when a PR is merged to the `main` branch.
```
Comment (abdulmth) [inline DEVELOPMENT.md:38]:
```suggestion
Releasing is trigged when a PR is merged to the `main` branch.
```

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+```bash
+ENDPOINT=peregrine ./scripts/setup.sh "build hill second flame trigger simple rigid cabbage phrase evolve final eight"
+
+```
+
+1. Change the base path in the `config.yaml` to
+
+```
+basePath: ./login-frontend/dist
```
Comment (ggera) [inline DEVELOPMENT.md:15]: This can be done via sed `sed -i 's|basePath: /srv|basePath: ./login-frontend/dist|' ./config.yaml`

Comment (abdulmth) [reply DEVELOPMENT.md:15]: sed commands depend on version and platform, let's let people do it manually.

File: DEVELOPMENT.md (hunk 1 above)
Comment (abdulmth) [reply DEVELOPMENT.md]: It's a random seed, I removed it.

**Review by ggera — APPROVED** [review]

**Pull request merged**